js/ui/html_editor/modules/toolbar.js

```diff
@@ -52,20 +52,14 @@ if(Quill) {
     const DIALOG_TABLE_FIELD_ROWS = 'dxHtmlEditor-dialogInsertTableColumnsField';
     const DIALOG_TABLE_CAPTION = 'dxHtmlEditor-dialogInsertTableCaption';
 
-    const TABLE_OPERATIONS = [
-        'insertTable',
-        'insertRowAbove',
-        'insertRowBelow',
-        'insertColumnLeft',
-        'insertColumnRight',
-        'deleteColumn',
-        'deleteRow',
-        'deleteTable'
-    ];
-
     const USER_ACTION = 'user';
     const SILENT_ACTION = 'silent';
 
+    const ICON_MAPPER = {
```

Comment:
why `mapper`, not `map`?

Reply:
I was thinking of something more global)